lib/main.js

```diff
@@ -38,7 +46,14 @@ export default {
       lintOnFly: true,
       lint: (activeEditor) => {
         const command = atom.config.get("linter-ruby.rubyExecutablePath");
+        const ignored = atom.config.get("linter-ruby.ignoredExtensions");
         const filePath = activeEditor.getPath();
+        const fileExtension = Path.extname(filePath).substr(1);
+
+        for (let extension of ignored) {
+          if (fileExtension === extension.trim()) return [];
```

Comment:
I think atom trims automatically for you when you use the type array. Can you please confirm? Other than that, awesome work on this PR :clap: